packages/web-api/src/methods.ts

```diff
@@ -521,6 +521,7 @@ export abstract class Methods extends EventEmitter<WebClientEvent> {
     access: bindApiCall<OAuthAccessArguments, OauthAccessResponse>(this, 'oauth.access'),
     v2: {
       access: bindApiCall<OAuthV2AccessArguments, OauthV2AccessResponse>(this, 'oauth.v2.access'),
+      exchange: bindApiCall<OAuthV2ExchangeArguments, OauthV2AccessResponse>(this, 'oauth.v2.exchange'),
```

Comment:
@misscoded Can you run `./scripts/generate-web-api-types.sh`? The command generates the response type for this endpoint. Also, it adds a few missing fields in other types.